File: 은지/LV1/가운데 글자 가져오기.py
def solution(s):
    # 단어의 길이가 홀수
    if len(s) % 2:
        answer = s[len(s)//2]
    # 단어의 길이가 짝수
    else:
        answer = s[len(s)//2-1:len(s)//2+1]
    return answer

File: 은지/LV1/가운데 글자 가져오기.js
function solution(s) {
  var answer = '';
  // 단어의 길이가 홀수
  if (s.length % 2) {
    answer = s[parseInt(s.length/2)]
  }
  // 단어의 길이가 짝수
  else {
    answer = s[parseInt(s.length/2)-1] + s[parseInt(s.length/2)]
  }
  return answer;
}